.changeset/strange-windows-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/utils': patch
+---
+
+remove `is-variation`
```

Comment:
What's the reason for removing it? Simply that it's not used anywhere?

Reply:
exactly. there was no use for it, neither in ui-kit or mc-frontend